### DISCUSSION TITLE
How I can run javascript function after updated property and rendered DOM

### DISCUSSION THREAD

**Discussion by nikUnit** [livewire/livewire#7289] [1. Help] [answered]
Hi! I have problem. How I can run javascript function after updated property and rendered DOM

I need an event that will happen after updating the property and rebuilding the DOM

**Comment by N1ebieski** [comment]
Probably related https://github.com/livewire/livewire/discussions/7062#discussioncomment-7278448

**Comment by nikUnit** [comment, marked as answer]
` ```
```
if (!empty($date_from) || !empty($date_to) || !empty($search)) {
    <hook-for-update style="display: none" id="<?= uniqid() ?>"><?= $date_from . $date_to . $search ?></hook-for-update>
}
```
      <script>
          document.addEventListener('livewire:init', () => {
              Livewire.hook('morph.added', ({
                  el
              }) => {
                  console.log(el.tagName)
                  if (el.tagName === "HOOK-FOR-UPDATE") {
                      console.log(el.tagName)
                      your javascript scripts
                  }
              })
          })
      </script>
````
This code i added below my component, when properties changed i call hook where choose tagname then run my js scripts
